.github/workflows/test.yml

```diff
@@ -103,7 +103,7 @@ jobs:
       with:
         submodules: recursive
     - name: Download LLVM
-      run: curl -sSL -o c:\llvm.zip https://github.com/hyperledger/solang/releases/download/v0.1.13/llvm13.0-win.zip
+      run: curl -sSL -o c:\llvm.zip https://github.com/hyperledger/solang/releases/download/v0.2.0/llvm13.0-win.zip
```

Comment:
These files are built during release, so usually we update these after release (not ideal)

Reply:
Shouldn't this work if first tag it and let the release job run? Since this is in the `test.yml`

Reply:
The release job is run in parallel to the test job, so it will work if we re-run it after the release job.

I don't know a way of making the test workflow depend on the release workflow, conditionally if there is a release workflow.

Reply:
As discussed on todays standup, to achieve this the first step would be to split our CI workflows to separate `build` from `test` jobs, so that early stage `build` artifacts can be used for tests but also as release assets. A further improvement would be to use something like bazel to gain the ability to "run" the CI locally.

But how should we proceed here, should we try it like this (with the re run) or should I undo these changes for a follow up (having it included here would be nicer IMO)?